impeller/renderer/backend/gles/buffer_bindings_gles.cc

```diff
@@ -415,7 +415,7 @@ std::optional<size_t> BufferBindingsGLES::BindTextures(
     /// If there is a sampler for the texture at the same index, configure the
     /// bound texture using that sampler.
     ///
-    const auto& sampler_gles = SamplerGLES::Cast(*data.sampler);
+    const auto& sampler_gles = SamplerGLES::Cast(**data.sampler);
```

Comment:
Does this count as making you a two star programmer? :)

Reply:
One day I'll sneak three stars in and become a legend.